.travis.yml

```diff
@@ -1,16 +1,11 @@
-language: cpp
-compiler:
-    - gcc
+sudo: required
+language: bash
+services:
+  - docker
+
 before_install:
-    # disable rvm, use system Ruby
-    - rvm reset
-    - wget https://raw.githubusercontent.com/yast/yast-devtools/master/travis-tools/travis_setup.sh
-    - sh ./travis_setup.sh -p "rake yast2-devtools yast2 yast2-core-dev libslp-dev" -g "rspec:3.3.0 yast-rake gettext simplecov coveralls:0.8.10"
+  - docker build -t yast-slp-image .
 script:
-    - rake check:syntax
-    - rake check:pot
-    - make -f Makefile.cvs
-    - make
-    - sudo make install
-    - make check
-
+  # the "yast-travis-cpp" script is included in the base yastdevel/cpp image
+  # see https://github.com/yast/docker-yast-cpp/blob/master/yast-travis-cpp
+  - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-slp-image yast-travis-cpp
```

Comment:
ok, i'm just curios when is yast-travis-cpp preferred over *-ruby one. As this module contains more then 50% of code written in ruby it is not that clear.